test/accessibility.spec.js

```diff
@@ -42,6 +42,7 @@ module.exports.addTests = function({testRunner, expect, FFOX}) {
         </select>
       </body>`);
 
+      await page.focus('[placeholder="Empty input"]');
```

Comment:
Isn't explicitly setting the focus defies the purpose of writing a test around it?